frontend/express/public/javascripts/countly/countly.common.js

```diff
@@ -232,7 +232,9 @@
         * @returns {string} decoded string
         */
         countlyCommon.decodeHtml = function(html) {
-            return (html + "").replace(/&amp;/g, '&');
+            var textArea = document.createElement('textarea');
```

Comment:
Hmmm not sure about this.

Is it fine @ar2rsawseen?

Reply:
I thought I could reuse it to decode html content that comes from the server (not only the ampersand operator). I guess I wanted to make it generic but I can keep it only for my use case since security is a concern.

Reply:
There are user activity data buckets that contain html unicode chars, e.g. &le;500.